--- origsrc/remind-03.01.13/src/calendar.c	2011-12-05 16:08:21.000000000 -0600
+++ src/remind-03.01.13/src/calendar.c	2013-10-15 01:12:56.870047300 -0500
@@ -11,6 +11,7 @@
 /***************************************************************/
 
 #include "config.h"
+#include "custom.h"
 
 #include <stdio.h>
 #include <string.h>
--- origsrc/remind-03.01.05/src/rem2ps.c	2008-01-30 20:37:40.000000000 -0600
+++ src/remind-03.01.05/src/rem2ps.c	2008-04-23 16:55:03.031250000 -0500
@@ -20,6 +20,7 @@
 
 #include <unistd.h>
 #include <stdlib.h>
+#include "custom.h"
 #include "rem2ps.h"
 
 #define NEW(type) (malloc(sizeof(type)))
--- origsrc/remind-03.01.05/src/types.h	2008-03-24 19:22:22.000000000 -0500
+++ src/remind-03.01.05/src/types.h	2008-04-23 16:55:03.046875000 -0500
@@ -11,6 +11,7 @@
 /***************************************************************/
 
 #include <limits.h>
+#include "custom.h"
 #include "dynbuf.h"
 
 /* Values */
